### ISSUE TITLE
Clicking links in the First Start Wizard asks you what application to open link with

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#2543]
### Short Summary

Tried this on Linux and Windows and it opens the same dialog so it must be a Komodo dialog but I've never seen it before and it's used regardless of whether or not I have the Komodo browser pref set.

### Steps to Reproduce

- Open the First Start Wizard
- CLick to last page
- click on any of the links

### Expected results

Komodo to open the link in a Browser

### Actual results

It opens this dialog always:
![capture](https://cloud.githubusercontent.com/assets/655154/25407097/f118a7ce-29bd-11e7-8a70-b1d7e94f4ed9.PNG)

**Comment by Naatan** [comment]
This is just how Komodo works; you have to configure a browser for this to do anything. If you run this on a profile that has a browser configured it shouldnt happen.

**Comment by cgchoffman** [comment]
> This is just how Komodo works; you have to configure a browser for thi

That's not true at all.  I've never seen this dialog and as I said I DID said the pref.
